apps/sensenet/src/components/AddDialog.tsx

```diff
@@ -0,0 +1,65 @@
+import Dialog, { DialogProps } from '@material-ui/core/Dialog/Dialog'
+import DialogContent from '@material-ui/core/DialogContent'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import { NewView } from '@sensenet/controls-react'
+import { GenericContent, Schema } from '@sensenet/default-content-types'
+import React, { useContext } from 'react'
+import { LocalizationContext, RepositoryContext } from '../context'
+import { LoggerContext } from '../context/LoggerContext'
+
+export interface AddDialogProps {
+  dialogProps: DialogProps
+  schema: Schema
+  parent: GenericContent
+}
+
+export const AddDialog: React.FunctionComponent<AddDialogProps> = ({ dialogProps, schema, parent }) => {
+  const localization = useContext(LocalizationContext).values.addButton
+  const repo = useContext(RepositoryContext)
+
+  const handleClose = () => dialogProps.onClose && dialogProps.onClose(null as any)
+  const logger = useContext(LoggerContext).withScope('AddDialog')
+
+  return (
+    <Dialog {...dialogProps}>
+      <DialogTitle> {localization.dialogTitle.replace('{0}', schema.DisplayName)}</DialogTitle>
+      <DialogContent>
+        <NewView
+          handleCancel={handleClose}
+          repository={repo}
+          contentTypeName={schema.ContentTypeName}
+          schema={schema}
+          path={parent.Path}
+          title=""
```

Comment:
Can this be omitted?
```suggestion
```

Reply:
At the moment is a bit hacky in the NewView - undefined will show the default title and it will be displayed twite (once in the ``<DialogTitle/>`` and once in the ``<NewView>``
A snippet from NewView:
```ts
{title !== undefined ? (
          <Typography variant="h5" gutterBottom={true}>
            {`New ${schema.schema.DisplayName}`}
          </Typography>
        ) : (
          title
        )}
```